conf/messages.en

```diff
@@ -1397,6 +1397,19 @@ fhlIncomeIntro.details.l1 = rental income
 fhlIncomeIntro.content.para2.individual = If your properties are jointly owned, only tell us about your share of these figures.
 fhlIncomeIntro.content.para2.agent = If your client's properties are jointly owned, only tell us about their share of these figures.
 
+ukrentaroom.income.totalIncomeAmount.title.individual=How much total rent a room income did you receive?
+ukrentaroom.income.totalIncomeAmount.heading.individual=How much total rent a room income did you receive?
+ukrentaroom.income.totalIncomeAmount.error.required.individual=Enter the the total rent a room income received
+ukrentaroom.income.totalIncomeAmount.error.twoDecimalPlaces.individual=The total amount of rent a room income received can only include pounds and pence, for example £600 or £600.20
+ukrentaroom.income.totalIncomeAmount.error.nonNumeric.individual=The total amount of rent a room income received can only include pounds and pence, for example £600 or £600.20
+ukrentaroom.income.totalIncomeAmount.error.outOfRange.individual=Enter an amount between £0 and £1,000,000,000
+
+ukrentaroom.income.totalIncomeAmount.title.agent=How much total rent a room income did your client receive?
+ukrentaroom.income.totalIncomeAmount.heading.agent=How much total rent a room income did your client receive?
+ukrentaroom.income.totalIncomeAmount.error.required.agent=Enter how much total rent a room income your client received
+ukrentaroom.income.totalIncomeAmount.error.twoDecimalPlaces.agent=The total amount of rent a room income received can only include pounds and pence, for example £600 or £600.20
+ukrentaroom.income.totalIncomeAmount.error.nonNumeric.agent=The total amount of rent a room income received can only include pounds and pence, for example £600 or £600.20
+ukrentaroom.income.totalIncomeAmount.error.outOfRange.agent=Enter an amount between £0 and £1,000,000,000
```

Comment:
Can you make it parameterised? You can refer in other places  

Enter an amount between £{} and £{}